packages/bolt-connection/src/connection-provider/connection-provider-direct.js

```diff
@@ -26,7 +26,7 @@ import {
 import { internal, error } from 'neo4j-driver-core'
 
 const {
-  constants: { BOLT_PROTOCOL_V4_0, BOLT_PROTOCOL_V3 }
+  constants: { BOLT_PROTOCOL_V4_0, BOLT_PROTOCOL_V3, BOLT_PROTOCOL_V4_4 }
```

Comment:
```suggestion
  constants: { BOLT_PROTOCOL_V3, BOLT_PROTOCOL_V4_0, BOLT_PROTOCOL_V4_4 }
```
or 
```suggestion
  constants: { BOLT_PROTOCOL_V4_4 , BOLT_PROTOCOL_V4_0, BOLT_PROTOCOL_V3 }
```
Feel free to ignore me :see_no_evil: